src/frontend/sharedComponents/Editor/PresetAndLocationView.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import {TouchableOpacity, View, Text, StyleSheet} from 'react-native';
+import {BLACK, COMAPEO_BLUE, LIGHT_GREY} from '../../lib/styles';
+import {Divider} from '../Divider';
+import {FormattedMessage, defineMessages} from 'react-intl';
+import LocationIcon from '../../images/Location.svg';
+import {FormattedCoords} from '../FormattedData';
+import {usePersistedSettings} from '../../hooks/persistedState/usePersistedSettings';
+
+const m = defineMessages({
+  searching: {
+    id: 'sharedComponents.EditScreen.PresetAndLocationView.seaching',
+    defaultMessage: 'Searching…',
+    description: 'Shown in new observation screen whilst looking for GPS',
+  },
+});
+
+export const PresetAndLocationView = ({
```

Comment:
this component is unnecessary. suggest having the `Editor` use the `PresetView` and `LocationView` directly so:

1. Create separate files for the `PresetView` and `LocationView`
2. Basically transfer the content of the body in this component to `Editor`

Reply:
https://github.com/digidem/comapeo-mobile/pull/416/commits/a04566a0651a9d93d55539fee881b1adf9277449